### PR TITLE
Screen size

### LINKED ISSUE #91: Screen sizing update
Figure out how to make it fit on phone, ipad, laptop, bigger screens

### PR DESCRIPTION
**Pull request by Rostammahabadi**
Added media queries to account for laptop screens and anything larger
Went through manually and checked each section to ensure the styling looked correct

Test are all passing

### CODE REVIEW

File: app/views/dashboard/show.html.erb [hunk 1]
```diff
@@ -43,7 +43,7 @@
 <br><br>
 <h2 class='text-center'>Compared Representatives</h2><br><br>
 <div class="container">
-  <div class="row">
+  <div class="row compared-rep">
```
Comment (arpariseau) [inline app/views/dashboard/show.html.erb:46]: With these CSS classes, probably not a bad thing to get granular

File: app/views/house_members/show.html.erb [hunk 2]
```diff
@@ -29,7 +29,7 @@
 </div>
 <% end %>
   <body>
-    <section class='rep-info' style="margin-left:225px;margin-top:25px;">
+    <section class='rep-info'>
```
Comment (arpariseau) [inline app/views/house_members/show.html.erb:32]: Removing in-line CSS. Thumbs up.

**Review by arpariseau — APPROVED** [review]

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -237,3 +232,78 @@
...
+    margin-left:225px;
+  }
+  .compared-rep{
+    margin-left:-.5em;
+  }
+  .bills-nav{
+    margin-right:400px;
+    font-size:20px;
+  }
+}
```
Comment (dborski) [inline app/assets/stylesheets/application.scss:309]: Interesting! Don't think I've ever said these words before, but CSS for the win!

**Review by dborski — APPROVED** [review]

File: app/views/house_members/show.html.erb (hunk 2 above)
Comment (AntonioJacksonII) [reply app/views/house_members/show.html.erb:32]: Agreed! Very nice.

**Review by AntonioJacksonII — APPROVED** [review]

**Pull request merged**